contracts/mocks/oracles/lido/LidoMock.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.6;
+import '../ISourceMock.sol';
+
+contract LidoMock is ISourceMock {
+    uint256 public price;
+
+    function set(uint256 price_) external override {
+        price = price_;
+    }
+
+    function getWstETHByStETH(uint256 _stETHAmount) external view returns (uint256) {
+        return (_stETHAmount * 10**18) / price;
```

Comment:
It's just a mock, but the `1e18` notation is cheaper in gas costs than `10**18`.